parachain/pallets/incentivized-channel/src/inbound/test.rs

```diff
@@ -259,3 +261,21 @@ fn test_submit_with_invalid_nonce() {
 		);
 	});
 }
+
+#[test]
+fn test_handle_fee() {
+	new_tester(SOURCE_CHANNEL_ADDR.into()).execute_with(|| {
+		let relayer: AccountId = Keyring::Bob.into();
+
+		let _ = Balances::deposit_creating(&SourceAccount::get(), 100000000000); // 10 DOT
+		let _ = Balances::deposit_creating(&TreasuryAccount::get(), Balances::minimum_balance());
```

Comment:
You use `resolve_creating` in `handle_fee`. So it should create this account if it doesn't exist, right? I'd add a test for that

Reply:
Yeah the E2E tests do verify that the treasury account is created and has the right amount deposited.